package.json

```diff
@@ -55,6 +55,7 @@
     "@babel/runtime-corejs2": "^7.1.5",
     "dequeue": "^1.0.5",
     "little-ds-toolkit": "^1.1.0",
+    "p-defer": "^1.0.0",
```

Comment:
Modules from sindresohrus are never transpiled to es5. This will break the es5 version

Reply:
I think should just drop es5 and legacy supports.